pkg/apis/operator/v1alpha1/tektontrigger_types.go

```diff
@@ -92,5 +92,7 @@ type TriggersProperties struct {
 // OptionalTriggersProperties defines the fields which are to be
 // defined for triggers only if user pass them
 type OptionalTriggersProperties struct {
-	DefaultServiceAccount string `json:"default-service-account,omitempty"`
+	DefaultServiceAccount string  `json:"default-service-account,omitempty"`
+	DefaultRunAsUser      *string `json:"default-run-as-user,omitempty"`
```

Comment:
we should not add it here, its a configmap thing and it can be done through options.

Reply:
we have already stopped adding more configs

Reply:
@savitaashture ah I just notice, it is applicable for kubernets too.
Will it work without `runAsUser` and `runAsGroup` in kubernetes?

Reply:
we should not add it in spec

we just remove those two values in openshift profile only

user should be able to configure through options in both profile

Reply:
okay 
got it
then in that case i want to set default value as `""` for both runAsUser and runAsGroup should i add under Options of Triggers  :thinking:

Reply:
just call the tranformer in openshift extension to set the default value